src/os_cloud_nodelet.cpp

```diff
@@ -72,53 +81,74 @@ class OusterCloud : public nodelet::Nodelet {
             tf_bcast.broadcast_transforms(info);
         } else {
             NODELET_INFO_STREAM(
-                "OusterCloud: dynamic transforms broadcast enabled wit "
+                "OusterCloud: dynamic transforms broadcast enabled with "
                 "broadcast rate of: "
                 << dynamic_transforms_rate << " Hz");
+            timer_.stop();
```

Comment:
Is this actually necessary? Mostly curious

Reply:
There was a bug of some sort
https://github.com/ouster-lidar/ouster-ros/issues/9#issuecomment-1311528903 that was causing an issue on for users (including me) that simply using the static broadcast is giving errors about out of sync. I wish to remove it personally :shrugging:

Reply:
It seems I maybe have misunderstood you question .. it probably isn't necessary since I am recreating the timer.